docs/docs/behaviors/macros.md

```diff
@@ -185,6 +145,70 @@ Macros use an internal queue to invoke each behavior in the bindings list when t
 
 To prevent issues with longer macros, you can change the size of this queue via the `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE` setting in your configuration, [typically through your `.conf` file](../config/index.md). For example, `CONFIG_ZMK_BEHAVIORS_QUEUE_SIZE=512` would allow your macro to type about 256 characters.
 
+## Parameterized Macros
+
+Macros can also be "parameterized", allowing them to be bound in your keymap with unique values passed into them, e.g.:
+
+```
+    raise_layer {
+        bindings = <&my_parameterized_macro A>
```

Comment:
```suggestion
        bindings = <&my_one_param_macro A>;
```
More consistent with the following section.

Reply:
Done